gnocchiclient/tests/functional/test_metric.py

```diff
@@ -51,6 +51,26 @@ def test_delete_several_metrics(self):
                                        "Metric %s does not exist (HTTP 404)" %
                                        metric2["id"])
 
+    def test_metric_listing(self):
+        apname = str(uuid.uuid4())
+        # PREPARE AN ARCHIVE POLICY
+        self.gnocchi("archive-policy", params="create "
+                     "-d granularity:1s,points:86400 %s" % apname)
+        # Create 10 metrics
+        for i in range(10):
+            self.gnocchi(
+                u'metric', params=u"create"
+                u" --archive-policy-name %s" % apname)
+
+        result = self.gnocchi(u'metric', params=u"list")
+        self.assertGreaterEqual(
+            len(self.parser.tables(result)[0]['values']), 10)
```

Comment:
wouldn't this always be >10? the limit is 1000 i think so your page link code is never really checked.